exercises/edward_brown/ex01_basics/include/Punctuation.h

```diff
@@ -0,0 +1,20 @@
+#ifndef EX1_PUNCTUATION
+#define EX1_PUNCTUATION
+#include <string>
+
+/** Determines if Candidate holds any of the following character values
+ *  `.,?'"!():`.
+ *
+ *  @param The character to test.
+ *  @return True if the Candidate is a punctuation mark, false if it is not.
+ */
+bool isPunctuation(char Candidate);
```

Comment:
Just looking at the code for exercise two, presumably this does not apply to pure virtual functions who's declarations are only in the header and which (unlike inline or unspecialized template functions which need to be accessible from the header) don't have a separate definition.